pkg/ingress/parser.go

```diff
@@ -243,6 +243,17 @@ func (c *controller) generateConfig() error {
 		si.AcceptProxy = true
 	}
 
+	if c.Ingress.SendProxyV2SSLCN() {
```

Comment:
@drf , instead of applying these annotations on the Ingress, I think these annotations should be applied to the backend Service and parsed from there. You should find other examples for this in the parser.

Reply:
you're right, too much copypasta. I'll fix this soon.